test/app-tap/datetime.test.lua

```diff
@@ -1867,6 +1868,51 @@ test:test("totable{}", function(test)
     end
 end)
 
+test:test('totable{} with timezone', function(test)
+    test:plan(6)
```

Comment:
I would rather declare it after `test_cases`, as `#test_cases * 2`.

Reply:
Fixed:

```diff
--- a/test/app-tap/datetime.test.lua
+++ b/test/app-tap/datetime.test.lua
@@ -1869,8 +1869,6 @@ test:test("totable{}", function(test)
 end)
 
 test:test('totable{} with timezone', function(test)
-    test:plan(6)
-
     local DEFAULT_TZOFFSET = 0
     local DEFAULT_TZ = ''
 
@@ -1904,6 +1902,8 @@ test:test('totable{} with timezone', function(test)
         },
     }
 
+    test:plan(#test_cases  * 2)
+
     for _, tc in pairs(test_cases) do
         local dt_table = date.new(tc.dt):totable()
         test:is(dt_table.tzoffset, tc.expected.tzoffset,
```